aat/build.gradle

```diff
@@ -42,6 +42,30 @@ configurations {
     cucumberRuntime.extendsFrom(functionalRuntime)
 }
 
+task smoke(type: Test) {
+    systemProperty "http.proxyHost", System.getProperty('http.proxyHost')
+    systemProperty "http.proxyPort", System.getProperty('http.proxyPort')
+    systemProperty "https.proxyHost", System.getProperty('https.proxyHost')
+    systemProperty "https.proxyPort", System.getProperty('https.proxyPort')
+
+    group = 'Delivery pipeline'
+    description = 'Executes smoke tests against an AAT CCD Case Activity API instance'
+    setTestClassesDirs(sourceSets.aat.output.classesDirs)
+    useJUnitPlatform {
+        includeTags 'smoke'
+    }
+    doLast {
+        generateCucumberReports.enabled = true
+        javaexec {
+            main = "uk.gov.hmcts.ccd.caseactivity.befta.CaseActivityBeftaMain"
+            classpath += configurations.cucumberRuntime + sourceSets.aat.runtimeClasspath + sourceSets.main.output + sourceSets.test.output
+            args = ['--plugin', "json:${rootDir}/target/cucumber.json", '--tags', '@Smoke', '--glue',
+                    'uk.gov.hmcts.befta.player', 'src/resources/features']
+        }
+    }
+    outputs.upToDateWhen { false }
+}
```

Comment:
this section would not contain some of these lines. pls check with data store example.